### DISCUSSION TITLE
How to add an event in Calendar app to Calendar subscriptions

### DISCUSSION THREAD

**Discussion by aloz77** [bitfireAT/icsx5#477] [Q&A]
I seem to be impossible to add a new event in Calendar app. _Calendar subscriptions_ can't be selected as calendar when adding an event. Tested with Samsung calendar app and Google calendar app. Can ICSx5 only provide the ICS calendar one direction (for viewing)?

ICSx5 is 2.2.5-gplay.

**Comment by ArnyminerZ** [comment]
Hi, calendar subscriptions are read-only, which means that you cannot create new events in them. At the same time, this means that ICSx⁵ only downloads data, doesn't have the hability to upload it.

In order to synchronize in both ways, you would need to use CalDAV, for which we have DAVx⁵ available 😉
